.storybook/frame-styles-decorator.js

```diff
@@ -8,9 +8,9 @@ const FrameStylesWrapper = fn => (
         <style jsx>{`
```

Comment:
I think we should drop this from Storybook entirely before we start to build components that rely on this existing in Applications.

Reply:
Agree